en/file_formats/geofence.md

```diff
@@ -0,0 +1,80 @@
+# GeoFence Plan File Format (Mission Protocol)
+
+GeoFence plan files are formatted using JSON, and can defined a cyclindrical geofence using parameters <!-- (autopilot-specific?) --> and/or an arbitrary polygon.
+
+<!--  doc originates from *QGroundControl Dev Guide*: https://dev.qgroundcontrol.com/en/file_formats/fence.html -->
+
+```
+{
+    "fileType": "GeoFence",
+    "groundStation": "QGroundControl",
+    "parameters": [
+        {
+            "compId": 1,
```

Comment:
What is compID?

Reply:
Component ID. Are you running an old build, I thought I had pulled out the parameters sections. It is no longer used.